File: Cap14CSharp/LiveSearchContext.cs
using System;
using Cap14CSharp;
using LINQToLive;

namespace Cap14CSharp
{
    public class LiveSearchContext
    {
        public Query<Resultado> Web;

        public Query<Resultado> News;

        public LiveSearchContext(String liveKey)
        {
            Web = new Query<Resultado>(new LiveProvider(liveKey, Live.Search.SourceType.Web));
            News = new Query<Resultado>(new LiveProvider(liveKey, Live.Search.SourceType.News));
        }
    }
}

File: Cap14VB/LiveSearchContext.vb
Imports System.Linq.Expressions
Imports Cap14VB.Live.Search

Public Class LiveSearchContext
  Public Web As Query(Of Resultado)

  Public News As Query(Of Resultado)

  Public Sub New(ByVal liveKey As String)
    Web = New Query(Of Resultado)(New LiveProvider(liveKey, Live.Search.SourceType.Web))
    News = New Query(Of Resultado)(New LiveProvider(liveKey, Live.Search.SourceType.News))
  End Sub
End Class
